website/static/js/osfHelpers.js

```diff
@@ -271,6 +271,21 @@ var mapByProperty = function(list, attr) {
 
 
 /**
+  * Return CSS colors and percentage of a progress bar based on a value.
+  * Used to return a value for password complexity
+  */
+
+var valueProgressBar = {
```

Comment:
Minor: I'm not sure this belongs here. The values (0-5) are really specific to password complexity, so it might make sense to just put this in `setPassword.js`.